README.md

```diff
@@ -13,20 +13,20 @@ It provides a simple, zero-cost, abstraction to clarify the _intent_ of your app
 
 ## Why Switch-HAL? Why not just use raw GPIO?
 
-Did you mean to drive that line high?  
-Or did you mean to _turn that LED off_?  
-Wait a second... is that LED active _high_?  
-Where's the schematic?  
+Did you mean to drive that line high?
+Or did you mean to _turn that LED off_?
+Wait a second... is that LED active _high_?
+Where's the schematic?
```

Comment:
The two spaces at the end of these lines was intentional. It’s for formatting. They insert a `<br>`

Reply:
Ah! Sorry about that, I didn't realize they were meaningful. My editor removed them automatically and I didn't think anything of it. Added them back.